sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -411,34 +437,42 @@ public async Task StartRecognizeCustomFormsFromUriThrowsForNonExistingContent(bo
             }
         }
 
-        [Test]
-        public void CreateFormTrainingClientFromFormRecognizerClient()
-        {
-            FormRecognizerClient client = CreateInstrumentedClient();
-            FormTrainingClient trainingClient = client.GetFormTrainingClient();
-            Assert.IsNotNull(trainingClient);
-        }
-
         /// <summary>
-        ///  For testing purposes, we are training our models using the client library functionalities.
-        ///  Please note that models can also be trained using a graphical user interface
-        ///  such as the Form Recognizer Labeling Tool found here:
-        ///  <a href="https://docs.microsoft.com/azure/cognitive-services/form-recognizer/quickstarts/label-tool"/>.
+        /// For testing purposes, we are training our models using the client library functionalities.
+        /// Please note that models can also be trained using a graphical user interface
+        /// such as the Form Recognizer Labeling Tool found here:
+        /// <a href="https://docs.microsoft.com/azure/cognitive-services/form-recognizer/quickstarts/label-tool"/>.
         /// </summary>
+        /// <param name="useLabels">If <c>true</c>, use a label file created in the &lt;link-to-label-tool-doc&gt; to provide training-time labels for training a model. If <c>false</c>, the model will be trained from forms only.</param>
+        /// <returns>The identifier of the trained model.</returns>
         private async Task<string> GetModelIdAsync(bool useLabels = false)
         {
-            FormTrainingClient trainingClient = InstrumentClient(new FormTrainingClient(_endpoint, _credential));
-            TrainingOperation trainedModel = await trainingClient.StartTrainingAsync(_containerUri, useLabels);
+            var trainingFiles = new Uri(TestEnvironment.BlobContainerSasUrl);
+            FormTrainingClient client = CreateInstrumentedFormTrainingClient();
+            TrainingOperation trainedModel;
+
+            // TODO: sanitize body and enable body recording here.
```

Comment:
Do we have an issue tracking this?

Reply:
I'm already working on it, but here it is: #11680